frontend_tests/node_tests/activity.js

```diff
@@ -615,7 +615,11 @@ test("initialize", ({override, mock_template}) => {
 
     let payload;
     override(channel, "post", (arg) => {
-        payload = arg;
+        if (payload === undefined) {
```

Comment:
This change is needed to make sure that the success function of first call when `want_redraw` is true is called for exisiting checksto pass.
(Since we are mocking `setTimeout` for calling the function immediately, function with `want_redraw` as true is executed before the call when `want_redraw` is false. But it is opposite in actual code due to the delay.)